helm/exporter-kube-scheduler/values.yaml

```diff
@@ -1,6 +1,6 @@
-# on what port are the metrics exposed by etcd
+# on what port are the metrics exposed by kube scheduler
 schedulerPort: 10251
-# for deployments that have etcd deployed outside of the cluster, list their adresses here
+# for deployments that have kube scheduler deployed outside of the cluster, list their adresses here
 endpoints: []
 # Are we talking http or https?
```

Comment:
Maybe add a pointer to https://github.com/kubernetes/kubernetes/issues/58983 here